backend/buildeR.API/Startup.cs

```diff
@@ -20,6 +22,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
+            services.AddDbContext<BuilderContext>(options => options.UseSqlServer(Configuration.GetConnectionString("BuilderDbConnection")));
```

Comment:
```suggestion
            services.AddDbContext<BuilderContext>(options => options.UseSqlServer(Configuration["BuilderDbConnection"]));
```

try this, because we are using env variables to store URLs/connection strings, etc.